bftengine/tests/simpleKVBC/include/KVBCInterfaces.h

```diff
@@ -13,218 +13,169 @@
-        // a time interval in milliseconds. represents how often the replica sends a status report to the other replicas.
-        // statusReportTimerMillisec > 0
-        uint16_t statusReportTimerMillisec;
-
-        // number of consensus operations that can be executed in parallel
-        // 1 <= concurrencyLevel <= 30
-        uint16_t concurrencyLevel;
-
-        // autoViewChangeEnabled=true , if the automatic view change protocol is enabled
-        bool autoViewChangeEnabled;
-
-        // a time interval in milliseconds. represents the timeout used by the  view change protocol (TODO: add more details)
-        uint16_t viewChangeTimerMillisec;
-
-        // maximum block size
-        uint32_t maxBlockSize;
-    };
-
-    struct ClientConfig
-    {
-        // F value - max number of faulty/malicious replicas. fVal >= 1
-        uint16_t fVal;
-
-        // C value. cVal >=0
-        uint16_t cVal;
-
-        // unique identifier of the client.
-        // clientId should also represent this client in ICommunication.
-        // In the current version, replicaId should be a number between N and  N+numOfClientProxies-1
-        // (N is the number replicas in the system. numOfClientProxies is part of the replicas' configuration)
-        uint16_t clientId;
-
-        // maximum reply size supported by this client
-        uint32_t maxReplySize;
-    };
-
-    /////////////////////////////////////////////////////////////////////////////
-    // Client proxy
-    /////////////////////////////////////////////////////////////////////////////
-
-    // Represents a client of the blockchain database
-    class IClient
-    {
-    public:
-        virtual Status start() = 0;
-        virtual Status stop() = 0;
-
-        virtual bool isRunning() = 0;
-
-        virtual Status invokeCommandSynch(const Slice command, bool isReadOnly, Slice& outReply) = 0;
-
-        virtual Status release(Slice& slice) = 0; // release memory allocated by invokeCommandSynch
-    };
-
-    // creates a new Client object
-    IClient* createClient(const ClientConfig& conf, bftEngine::ICommunication* comm);
-
-    // TODO: Implement:
-    //  // deletes a Client object
-    //  void release(IClient* r);
-
-    /////////////////////////////////////////////////////////////////////////////
-    // Replica
-    /////////////////////////////////////////////////////////////////////////////
-
-    // Represents a replica of the blockchain database
-    class IReplica
-    {
-    public:
-        virtual Status start() = 0;
-        virtual Status stop()  = 0;
-        virtual ~IReplica(){}
-        enum class RepStatus // status of the replica
-        {
-            UnknownError = -1,
-            Ready                = 0,
-            Starting,
-            Running,
-            Stopping,
-        };
-
-        // returns the current status of the replica
-        virtual RepStatus getReplicaStatus() const = 0;
+using concordUtils::Status;
+using concordUtils::Sliver;
```

Comment:
I guess I missed these two the first time. There should not be `using` directives in header files.

Reply:
I'll fix that in a future commit. Thanks!